astro/src/content/blog/weroad-fusionauth.mdx

```diff
@@ -0,0 +1,107 @@
+---
+publish_date: 2025-06-09
+title: "Enterprise-Ready: How WeRoad Scaled Their Authentication Architecture with FusionAuth"
+description: This case study explores how WeRoad, a rapidly growing European travel company, successfully migrated from an expensive third-party authentication provider to a self-hosted solution that gave them complete control over their user data and authentication infrastructure.
+authors: Brad McCarty
+image: /img/blogs/weroad/weroad.png
+categories: Customers
+tags: weroad, customers, m2m, passwordless, data sovereignty
+excerpt_separator: "{/* more */}"
+---
+
+WeRoad is a European adventure travel company founded in 2017 that specializes in organizing group travel experiences for adventurous travelers. Based in Europe with operations across multiple countries including the UK, WeRoad has grown rapidly since its founding, expanding from a small startup to an established company with approximately 200 employees, including a technical team of around 50 engineers.
+
+[WeRoad](https://www.weroad.com/) creates unique travel experiences that connect like-minded explorers, organizing adventures in various destinations across the globe. Their focus on community-building and authentic experiences has helped them carve out a distinct niche in the competitive travel industry.
+
+## The Challenge
+
+As WeRoad rapidly expanded, their authentication needs grew increasingly complex. The DevOps team, led by Roberto Dedoro, faced several authentication challenges:
+
+- Authentication for Multiple Applications: They needed a [single sign-on](/sso) solution that would work across multiple applications
+- Social Login Support: The ability for users to [authenticate using Google](/glossary/social-sign-on-social-login) and potentially other social providers
+- Mobile App Authentication: Supporting authentication for their newly launched Android and iOS applications
+- Machine-to-Machine Authentication: Secure tokens for backend-to-backend services via [M2M](/feature/machine-to-machine)
+- Passwordless Options: Support for magic links and other [passwordless authentication](/passwordless) methods
+- Data Sovereignty: As a European company subject to GDPR, maintaining [control over user data](/platform/self-hosting) was essential
+
+![](/img/blogs/weroad/weroad-sso-diagram.png)
+
+Additionally, WeRoad was facing significant cost increases with their previous authentication provider, Auth0, which was becoming unsustainable as they scaled.
+
+## **Why FusionAuth?**
```

Comment:
No need for the bold, this is already an H2. 

We should remove the `**` values around all the headers, as they don't do anything.